spec/features/recipes/show_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe 'Recipes Show Page' do 
+  # User Story 2 - Recipes Show
+
+  # As a visitor,
+  # When I visit '/recipes/:id',
+  # Then I see the recipe's name, complexity and genre,
+  # and I see a list of the names of the ingredients for the recipe.
+
+  describe 'Show Page' do 
+    it 'shows the recipe info ' do 
+    recipe = Recipe.create!(name: "Crispy Pork Belly", complexity: 3, genre: "Awesome")
+
+    visit "/recipes/#{recipe.id}"
+
+    expect(page).to have_content("Crispy Pork Belly")
+    expect(page).to have_content("Recipe Complexity: 3")
+    expect(page).to have_content("Recipe Genre: Awesome")
+  end
+
+    it 'shows the ingredients used in recipe' do 
+      recipe = Recipe.create!(name: "Crispy Pork Belly", complexity: 3, genre: "Awesome")
+      ingredient_1 = recipe.ingredients.create!(name: "Pork Belly", cost: 11)
+      ingredient_2 = recipe.ingredients.create!(name: "Salt", cost: 100)
```

Comment:
Good object setup overall. Might be good to have a 2nd recipe with no ingredients, or only 1, just to be able to set up different scenarios in your tests to get better quality coverage.